.gitignore

```diff
@@ -54,3 +54,5 @@ internal/mode/static/nginx/modules/coverage
 
 # SSH config files
 *.ssh
+
+config/base/deploy.yaml
```

Comment:
What's this about?

Reply:
it's the file used for manifests generation, added a comment